test/byContract/IexecPoco/06_claim.test.ts

```diff
@@ -307,6 +306,126 @@ describe('Poco', async () => {
         await expect(iexecPoco.connect(anyone).claim(taskId)).to.be.reverted;
     });
 
+    describe('Claim array', function () {
+        it('Should claim array', async function () {
+            const volume = 3;
+            const { orders } = buildOrders({
+                assets: ordersAssets,
+                requester: requester.address,
+                prices: ordersPrices,
+                volume,
+            });
+            const { dealId, startTime } = await signAndMatchOrders(orders);
+            const taskIds = [];
+            for (let taskIndex = 0; taskIndex < volume; taskIndex++) {
+                taskIds.push(getTaskId(dealId, taskIndex));
+                await iexecPoco
+                    .connect(scheduler)
+                    .initialize(dealId, taskIndex)
+                    .then((tx) => tx.wait());
+            }
+            // Mine empty block so timestamp is accurate when static call is made
+            await time.setNextBlockTimestamp(startTime + maxDealDuration).then(() => mine());
```

Comment:
Optional suggestion to make it more readable (no need to chain here since they are independent).
```suggestion
            // Mine empty block so timestamp is accurate when static call is made
            await time.setNextBlockTimestamp(startTime + maxDealDuration);
            await mine();
```